dev-utils/test-config.js

```diff
@@ -112,7 +112,7 @@ function getAppiumBrowsersForKarma() {
       browserName: 'Browser',
       appiumVersion: '1.20.2',
       deviceName: 'Android Emulator',
-      platformVersion: '5.1'
+      platformVersion: '6.0'
```

Comment:
Increased because 5.1. was not even compatible with Object.defineProperty which now is being used in the page visibility tests

In the issue where we will drop support to IE11 we should check the version of this browser too